### ISSUE TITLE
SPOHomeSite

### ISSUE THREAD

**Issue by andikrueger** [microsoft/Office365DSC#281]
### Description
New Resource for getting and setting the SPO Home Site.

Get: https://docs.microsoft.com/en-us/powershell/module/sharepoint-pnp/get-pnphomesite?view=sharepoint-ps
Set: https://docs.microsoft.com/en-us/powershell/module/sharepoint-pnp/set-pnphomesite?view=sharepoint-ps

### Proposed properties
URL - The url of the site to set as the home site

**Comment by NikCharlebois** [comment]
@andikrueger is this your way of logging that you are working on this resource? :)

**Comment by andikrueger** [comment]
Yeah, sure. You can assign me on this one.

**Comment by andikrueger** [comment]
I did some test with the Get and Set Methods and am not sure, if it works correctly. During my test, It took hours to replicate changes and setting a new home site just by running the set method did not succeed. I will submit my PR regardless.

**Comment by NikCharlebois** [comment]
Ok, and when you say it take a long time to replicate, is that only at the UI level? Meaning, if you run the Set and then the Get, do you get the new values returned at the PoSh level right away?

**Comment by andikrueger** [comment]
This is my output from a recent PoSh. There is no instance feedback within PoSh too.

```powershell
PS C:\Temp> Set-SPOHomeSite -HomeSiteUrl https://tenant.sharepoint.com/sites/ContosoWeb1
Bestätigung
Before you set a Home site, make sure you review the documentation at https://aka.ms/homesites. Continue?
[J] Ja [N] Nein [H] Anhalten [?] Help (default is "Ja"): j

Value
-----
Die Startsite wurde auf "https://tenant.sharepoint.com/sites/ContosoWeb1" festgelegt. Es kann etwas dauern, bis die Änderung angewendet wird. Details finden Sie unter "aka.ms/homesites".


PS C:\Temp> Get-SPOHomeSite

Value
-----
https://tenant.sharepoint.com/sites/Work
```

**Comment by NikCharlebois** [comment]
@andikrueger I would just submit the PR so that we can start the reviewing process. I will run some tests on my end in parallel.

**Comment by andikrueger** [comment]
I will submit a PR by tomorrow.

**Comment by andikrueger** [comment]
Closed by #344
